packages/relay/tests/lib/services/transactionService/transactionService.spec.ts

```diff
@@ -208,4 +219,24 @@ describe('Transaction Service', function () {
       );
     });
   });
+
+  describe('calculateTxRecordChargeAmount', () => {
+    it('Should execute calculateTxRecordChargeAmount() to get the charge amount of transaction record', () => {
+      const mockedExchangeRateIncents = 12;
+      const txQueryCostInCents = constants.TX_RECORD_QUERY_COST_IN_CENTS;
+      const hbarToTinybar = Hbar.from(1, HbarUnit.Hbar).toTinybars().toNumber();
+      const expectedTxRecordAmount = Math.round((txQueryCostInCents / mockedExchangeRateIncents) * hbarToTinybar);
+
+      const mockedExchangeRate = {
+        hbars: 30000,
+        cents: 164330,
+        expirationTime: new Date(),
+        exchangeRateInCents: mockedExchangeRateIncents,
+      } as ExchangeRate;
+
+      const txRecordChargedAmount = transactionService.calculateTxRecordChargeAmount(mockedExchangeRate);
```

Comment:
Please, duplicate the calculation here instead of using the actual logic. Otherwise, introducing a bug inside the actual method would be reflected here and wont be caught by the tests.